### PR TITLE
ANDROID-13476 make logger great again

### PR DESCRIPTION
**Pull request by jdelga**
ANDROID-13476

Logger project can't compile due to an Android Gradle Plugin incompatible with recent Android Studio versions

I have also updated useful properties such as jvm
Most changed are automatic updates of the gradle wrapper files

Before:

logger_upgrade_before

After:

logger_upgrade_after

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -23,18 +23,16 @@ android {
...
+        jvmTarget = '11'
     }
+    namespace 'com.telefonica.androidlogger.app'
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.3.2'
     implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'com.google.android.material:material:$support_version'
```
Comment (jdelga) [inline app/build.gradle:38]: Unused

**Review by jeslat — APPROVED** [review]
👍

File: library/src/enabled/AndroidManifest.xml [hunk 2]
```diff
@@ -1,7 +1,5 @@
 <manifest
-		xmlns:android="http://schemas.android.com/apk/res/android"
-		package="com.telefonica.androidlogger"
```
Comment (yamal-alm) [inline library/src/enabled/AndroidManifest.xml:3]: don't we need to add the `namespace` in library/build.gradle just like in app module?

Comment (jdelga) [reply library/src/enabled/AndroidManifest.xml:3]: You can find it here: https://github.com/Telefonica/android-logger/pull/13/files#diff-1cfc99164a9e87fb25dc437e1caa400c8144dcec3cf57c8766f7498e5957028dR33

Comment (yamal-alm) [reply library/src/enabled/AndroidManifest.xml:3]: yeah, I'm blind 👨🏻‍🦯

**Review by yamal-alm — APPROVED** [review]
thanks 👏

**Pull request merged**